templates/taxi/car_detail.html

```diff
@@ -3,13 +3,15 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
+      {% if user in car.drivers.all %}
```

Comment:
Ensure that the `car.drivers.all` queryset is evaluated correctly. Using `all` in the template might lead to performance issues if the list of drivers is large. Consider optimizing this if necessary.